bindings/python/pinocchio/visualize/meshcat_visualizer.py

```diff
@@ -739,7 +739,7 @@ def loadMeshFromFile(self, geometry_object):
         # Get file type from filename extension.
         _, file_extension = os.path.splitext(geometry_object.meshPath)
         if file_extension.lower() == ".dae":
-            obj = DaeMeshGeometry(geometry_object.meshPath)
+            obj = mg.DaeMeshGeometry.from_file(geometry_object.meshPath)
```

Comment:
We should remove our implementation of DaeMeshGeometry.

Also the following code can be removed :

```python
# DaeMeshGeometry
import xml.etree.ElementTree as Et
import base64
```